test-result-summary-client/src/Build/GitNewIssue.jsx

```diff
@@ -25,9 +26,9 @@ export default class GitNewissue extends Component {
         const { testId, testName, duration, buildId, buildName,
             buildUrl, machine, buildTimeStamp, javaVersion, testResult } = getParams(this.props.location.search);
 
-        let firstSeenFailure = null;
+        let firstSeenFailure, successBeforeFailure, gitDiffLinks = null;
```

Comment:
`successBeforeFailure` and `gitDiffLinks` should be defined within `if (testResult === 'FAILED') {` block. They are not used outside of the if block.